tests/plugins/prithvi_io_processor_plugin/prithvi_io_processor/prithvi_processor.py

```diff
@@ -220,7 +223,8 @@ def load_image(
                         julian_day, "%m%d").timetuple().tm_yday)
                 temporal_coords.append([year, julian_day])
         except Exception as e:
-            print(f"Could not extract timestamp for {file} ({e})")
+            logger.exception("Could not extract timestamp for %s (%s)", file,
+                             str(e))
```

Comment:
```suggestion
            logger.exception("Could not extract timestamp for %s", file)
```

The exception stack trace is logged automatically under `logger.exception` so no need to include it in the string again

Reply:
I suspected this, but I found both on the code base. I'm removing it now.